pkg/common/cns-lib/vsphere/utils.go

```diff
@@ -396,3 +405,36 @@ func isVsan67u3Release(ctx context.Context, m *defaultVirtualCenterManager, host
 	log.Debugf("vCenter version is :%q", vc.Client.Version)
 	return vc.Client.Version == cns.ReleaseVSAN67u3, nil
 }
+
+// IsvSphereVersion70U3orAbove checks if specified version is 7.0 Update 3 or higher
+// The method takes aboutInfo{} as input which contains details about
+// VC version, build number and so on.
+// If the version is 7.0 Update 3 or higher, the method returns true, else returns false
+// along with appropriate errors during failure cases
+func IsvSphereVersion70U3orAbove(ctx context.Context, aboutInfo types.AboutInfo) (bool, error) {
+	log := logger.GetLogger(ctx)
+	items := strings.Split(aboutInfo.Version, ".")
+	version := strings.Join(items[:], "")
+	// Convert version string to string, Ex: "7.0.3" becomes 703, "7.0.3.1" becomes 703
+	if len(version) >= 3 {
+		vSphereVersionInt, err := strconv.Atoi(version[0:3])
+		if err != nil {
+			msg := fmt.Sprintf("error while converting version %q to integer, err %+v", version, err)
+			log.Errorf(msg)
+			return false, errors.New(msg)
+		}
+		vSphere70u3VersionStr := strings.Join(strings.Split(VSphere70u3Version, "."), "")
+		vSphere70u3VersionInt, err := strconv.Atoi(vSphere70u3VersionStr[0:3])
```

Comment:
Why not just store this as a constant too?

Reply:
Makes sense.